managedapps/legacyresource/error.go

```diff
@@ -0,0 +1,48 @@
+package legacyresource
```

Comment:
This legacyresource was extracted from e2e-harness. Let's keep it for now. We can refactor later.